app/assets/stylesheets/admin/calendar/_calendar.scss

```diff
@@ -25,9 +25,8 @@
       font-size: 1em;
       line-height: 1.2em;
 
-      &.fa-link {
-        color: $gray3;
-        padding-right: .25em;
+      span {
+        font-family: "Open Sans",sans-serif;
```

Comment:
I think this should be variable, shouldn't it? If these are the main fonts for the admin panel and there is not already of variable, we should make one. Or if this is some special style,We could have a variable for that too.

What do you think?

Reply:
I'll look into it. It sounds sensible to make the fonts a variable.